funding/manager.go

```diff
@@ -3628,6 +3607,46 @@ func (f *Manager) handleChannelReady(peer lnpeer.Peer,
 		f.barrierMtx.Unlock()
 	}()
 
+	// Before we can add the channel to the peer, we'll need to ensure that
```

Comment:
This piece seems to be very dedicated to doing one thing and I think it can fit into its own method, also easier for unit testing(future, hopefully...)